src/org/opendatakit/briefcase/ui/BriefcaseCLI.java

```diff
@@ -1,320 +0,0 @@
-/*
```

Comment:
This class has been replaced by the classes in the `org.opendatakit.briefcase.operations` package.